.gitignore

```diff
@@ -0,0 +1,4 @@
+softhsm.conf
```

Comment:
I see this file referenced in init-softhsm.sh, but I don't see where it is populated. Where does it come from, and why is it ignored?

Reply:
It'll get created by `test.sh`. It's ignored because at creation time it has the full path including username like /home/phil/whatever/softhsm/.